console/src/components/Services/EventTrigger/Add/AddTrigger.js

```diff
@@ -48,13 +50,15 @@ class AddTrigger extends Component {
     if (this.props.serverVersion) {
       this.checkSemVer(this.props.serverVersion).then(() => {
         this.checkWebhookEnvSupport(this.props.serverVersion);
+        this.checkRetryTimeoutSupport(this.props.serverVersion);
       });
     }
   }
   componentWillReceiveProps(nextProps) {
     if (nextProps.serverVersion !== this.props.serverVersion) {
       this.checkSemVer(nextProps.serverVersion).then(() => {
         this.checkWebhookEnvSupport(nextProps.serverVersion);
+        this.checkRetryTimeoutSupport(this.props.serverVersion);
```

Comment:
The above statement uses nextProps.serverVersion and this one uses this.props.serverVersion.
Which one is valid?